PathWeaver/ThreeBallPartnerAuton/PathWeaver/pathweaver.json

```diff
@@ -0,0 +1,9 @@
+{
+  "lengthUnit": "Meter",
+  "exportUnit": "Always Meters",
+  "maxVelocity": 3.0,
+  "maxAcceleration": 3.0,
```

Comment:
I think acceleration should be less here? Should probably base it off of the 5 ball auton velocity and acceleration